### PR TITLE
[MM-51797] Add bullet point on calls transmission security

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR adds a bullet point to our transmission security section to address the security of calls.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-51797

### CODE REVIEW

**Comment by streamer45** [comment]
@jupenur Would be great if you could give this a quick review to make sure everything is sound from a security perspective. Let me know :)

File: source/about/security.rst [hunk 1]
```diff
@@ -35,6 +35,12 @@ Transmission security
 ~~~~~~~~~~~~~~~~~~~~~~
 
 - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet.
+- Connections to calls are secured with a combination of:
```
Comment (jupenur) [inline source/about/security.rst:38]: 0/5 is it clear enough that this is an exception to the above?

**Review by jupenur — APPROVED** [review]
Apologies for the delay, I was on PTO. Just one small comment, otherwise LGTM :)

File: source/about/security.rst (hunk 1 above)
Comment (cwarnermm) [reply source/about/security.rst:38]: @streamer45 ^

File: source/about/security.rst [hunk 2]
```diff
@@ -35,6 +35,12 @@ Transmission security
 ~~~~~~~~~~~~~~~~~~~~~~
 
 - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet.
```
Comment (cwarnermm) [inline source/about/security.rst:37]:
```suggestion
- Mattermost supports TLS encryption using AES-256 with 2048-bit RSA between Mattermost client applications and the Mattermost server across both LAN and internet.
```

**Pull request merged**